druid/src/text/text_input.rs

```diff
@@ -0,0 +1,96 @@
+use super::Movement;
```

Comment:
can we get a license header and a `//! short module comment` here?

Reply:
Done